src/app/application.cpp

```diff
@@ -278,7 +279,7 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
         program.replace("%T", torrent->currentTracker());
         program.replace("%I", torrent->hash());
 
-        QProcess::startDetached(program);
+        std::thread ([program]() { return system(program.toStdString().c_str()); }).detach();
```

Comment:
@evsh 
>No matter exits app or not, creating unmaintained threads is a bad style. Think about debugging, for example and all the additional threads and stacktraces. 

I don't understand. Who debugs an external script running via qbt? don't you debug it directly in the shell (before hooking up to qbt)?

>And a quote from the man page for system()

There is no simple way of executing shell cmds without platform dependent code. If we want cross-platform code, the available options is limited.